themes/bootstrap5/js/fines.js

```diff
@@ -0,0 +1,169 @@
+/*global VuFind */
+VuFind.register('fines', function fines() {
+  const CHECKBOX_ITEM_SELECTOR = 'form#online_payment_form .js-select-item';
+  const CHECKBOX_ALL_SELECTOR = 'form#online_payment_form .js-select-all';
+
+  let paySelectedDefaultText;
+
+  /**
+   * Get the whole part from currency in cents
+   * @param {number} currency Number to divide by 100
+   * @returns {number} Currency without digits
+   */
+  function getWhole(currency)
+  {
+    return Math.trunc(currency / 100);
+  }
+
+  /**
+   * Get the fraction part from currency in cents padded to two characters
+   * @param {number} currency Currency the get fraction from
+   * @returns {string} Fraction of the currency
+   */
+  function getFraction(currency)
+  {
+    var fraction = String(currency % 100);
+    while (fraction.length < 2) {
+      fraction += '0';
+    }
+    return fraction;
+  }
+
+  /**
+   * Format currency according to a template where 11 is whole and 22 is fraction
+   * @param {number} currency Currency to use in template
+   * @param {string} template Template string to insert into
+   * @returns {string} Formatted amount
+   */
+  function formatAmount(currency, template)
+  {
+    // Prevent cases where whole with 22 or 122 gets overwritten with the fraction
+    return template.replace('11', '{whole}')
+      .replace('22', '{fraction}')
+      .replace('{whole}', getWhole(currency))
+      .replace('{fraction}', getFraction(currency));
+  }
+
+  /**
+   * Set the aria-live attribute for an element
+   * @param {Element} element Element to set aria live into
+   * @param {string} politeness Aria live value
```

Comment:
Do you need to clarify that an empty value removes the attribute?

```suggestion
   * @param {string} politeness Aria live value (empty string to remove attribute)
```